### PR TITLE
Adding ITA translations

### PR DESCRIPTION
**Pull request by lonasozo**
## Explanation
I've seen there are ***a lot*** of strings not translated in Italian, adding them slowly. 
Created this drafted PR so that anyone who wants to help is welcome, if there is some specs about strings not translated (maybe there is some reason for some specific Strings), please let me know.

I'm going on by checking each string on __locales/en_

### CODE REVIEW

**Comment by lonasozo** [comment]
It's not clear to me why "ci/circleci: test-lint" fails, some suggestions?

**Comment by lonasozo** [comment]
Guys I'm working on my free time but not have too much free time at moment. 
I've seen that branch need to be updated and conflicts happens often, so I'm opening this PR, it can be already merged and has a lot of translations.

I'll still continue working on new translations as fast as I can

**Comment by brad-decker** [comment]
> Guys I'm working into it on my free time but not have too much free time at moment.
I've seen that branch need to be updated and conflicts happens often, so I'm opening this PR, it can be already merged and has a lot of translations.

First off, @lonasozo , I am so sorry I missed this Pull Request. I thought I had went through and started replying to all of the translation prs now that the translation system is in place. Thank you for doing this incredible work. 

So our lint tool is complaining because the file isn't in the right format  (keys are automatically sorted in this file by running `yarn verify-locales:fix`, which *should* help reduce the churn (+/- lines of change) and make this significantly easier to review. I'm going to do that for you now but wanted to arm you with that knowledge for the future.

**Review by brad-decker — DISMISSED** [review]
and it LGTM i spot checked a few translations. I'll get another reviewer to take a look too

**Comment by lonasozo** [comment]
Thank you @brad-decker ! Clear now.
I'll continue working on that into a new PR as soon as I'll have some free time. 
In the while good to know this will be merged

**Comment by brad-decker** [comment]
@lonasozo i made a slight mistake. so `yarn verify-locales:fix` will remove keys that have been removed from the application (those no longer used). `yarn lint:fix` will sort the translation files alphabetically. Both have been done on this PR and i'm re-approving.

**Review by brad-decker — DISMISSED** [review]

File: app/_locales/it/messages.json
Comment (lonasozo) [reply app/_locales/it/messages.json:24]: Mi sa di sì 😅 fixo appena ho pc sotto mano e controllo se ho lasciato altri "wallet" in giro

**Comment by lonasozo** [comment]
Guys I've fixed but got issues to run yarn to lint code, I'll check on that, I'll be glad if someone can lint it for me this time

**Comment by brad-decker** [comment]
@lonasozo great work fixing lint! I'm looking into the e2e test failures for you.

**Review by brad-decker — DISMISSED** [review]

**Comment by brad-decker** [comment]
@Gtonizuka ready for another look :)

**Comment by lonasozo** [comment]
Done guys, still test-e2e-chrome and test-e2e-firefox. 
How can I fix them?

**Comment by brad-decker** [comment]
@lonasozo those aren't your fault, we have fixed some flaky e2e tests on other prs so i'm just going to update your branch and hope for the best. 😅

**Comment by brad-decker** [comment]
@Gtonizuka can you go ahead and do another review pass while we wait for CI? 🙏🏻 Thanks for all the assistance here.

**Comment by lonasozo** [comment]
Re-updated branch

**Review by brad-decker — APPROVED** [review]

**Review by Gtonizuka — APPROVED** [review]
let's do it @lonasozo @brad-decker

**Pull request merged**
